options/locale/locale_en-US.ini

```diff
@@ -1291,11 +1291,11 @@ issues.remove_label = removed the %s label %s
 issues.remove_labels = removed the %s labels %s
 issues.add_remove_labels = added %s and removed %s labels %s
 issues.add_milestone_at = `added this to the <b>%s</b> milestone %s`
-issues.add_project_at = `added this to the <b>%s</b> project %s`
+issues.add_project_at = `added this to the <span class="tooltip" data-content="%s"><b>%s</b></span> project %s`
 issues.change_milestone_at = `modified the milestone from <b>%s</b> to <b>%s</b> %s`
-issues.change_project_at = `modified the project from <b>%s</b> to <b>%s</b> %s`
+issues.change_project_at = `modified the project from <span class="tooltip" data-content="%s"><b>%s</b></span> to <span class="tooltip" data-content="%s"><b>%s</b></span> %s`
 issues.remove_milestone_at = `removed this from the <b>%s</b> milestone %s`
-issues.remove_project_at = `removed this from the <b>%s</b> project %s`
+issues.remove_project_at = `removed this from the <span class="tooltip" data-content="%s"><b>%s</b></span> project %s`
```

Comment:
Two things:

1. The `tooltip` has been dropped. Use `data-tooltip-content` instead.
2. We should avoid constructing HTML in translation strings, translators may break them.
    * To make the locale system stable, I guess we should prepare the HTML in HTML, then the tempate only do :
    `added this to the %s project %s`